### PR TITLE
26865 Removed deprecations from quiver.py

### PR DESCRIPTION
**Pull request by wemi3**
## PR summary
This PR removes the depcrecated code quiver_doc and barbs_doc during the version 3.7 from quiver.py for the issue https://github.com/matplotlib/matplotlib/issues/26865

## PR checklist

- [ N/A] "closes #0000" is in the body of the PR description to [link the related issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- [N/A ] new and changed code is [tested](https://matplotlib.org/devdocs/devel/testing.html)
- [N/A ] *Plotting related* features are demonstrated in an [example](https://matplotlib.org/devdocs/devel/document.html#write-examples-and-tutorials)
- [N/A ] *New Features* and *API Changes* are noted with a [directive and release note](https://matplotlib.org/devdocs/devel/coding_guide.html#new-features-and-api-changes)
- [N/A ] Documentation complies with [general](https://matplotlib.org/devdocs/devel/document.html#write-rest-pages) and [docstring](https://matplotlib.org/devdocs/devel/document.html#write-docstrings) guidelines

### CODE REVIEW

File: doc/api/next_api_changes/removals/26918-EW.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+``quiver_doc``
+~~~~~~~~~~
+This attribute of ``Quiver``\s has been removed.
```
[the remark this answers is not in the document]
Comment (wemi3) [reply doc/api/next_api_changes/removals/26918-EW.rst:3]: Thanks, will do!

**Review by ksunden — APPROVED** [review]

File: lib/matplotlib/quiver.py [hunk 2]
```diff
@@ -720,10 +720,6 @@ def _h_arrows(self, length):
             np.copyto(Y, Y1, where=tooshort)
         # Mask handling is deferred to the caller, _make_verts.
         return X, Y
-
-    quiver_doc = _api.deprecated("3.7")(property(lambda self: _quiver_doc))
-
-
 _barbs_doc = r"""
```
Comment (QuLogic) [inline lib/matplotlib/quiver.py:723]: These two blank lines should be kept.
```suggestion


_barbs_doc = r"""
```

File: doc/api/next_api_changes/removals/26918-EW.rst [hunk 3]
```diff
@@ -0,0 +1,3 @@
+``Quiver.quiver_doc and Barbs.barbs_doc``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+... are deprecated. These are the doc-string and should not be accessible as a named class member.
```
Comment (oscargus) [inline doc/api/next_api_changes/removals/26918-EW.rst:3]:
```suggestion
... are removed. These are the doc-string and should not be accessible as a named class member.
```

File: doc/api/next_api_changes/removals/26918-EW.rst [hunk 4]
```diff
@@ -0,0 +1,3 @@
+``Quiver.quiver_doc and Barbs.barbs_doc``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (QuLogic) [inline doc/api/next_api_changes/removals/26918-EW.rst:2]:
```suggestion
``Quiver.quiver_doc`` and ``Barbs.barbs_doc``
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Review by QuLogic — APPROVED** [review]

**Comment by QuLogic** [comment]
Thanks @wemi3! Congratulations on your first PR to Matplotlib :tada: We hope to hear from you again.

**Pull request merged**